src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -70,19 +70,19 @@
     private ProgressBar uploadProgress;
 
     @FXML
-    private ImageView feedbackButton;
+    public ImageView feedbackButton;
 
     @FXML
-    private ImageView inviteButton;
+    public ImageView inviteButton;
 
     @FXML
     private ImageView configButton;
 
     @FXML
-    private ImageView faqButton;
+    public ImageView faqButton;
 
     @FXML
-    private ImageView infoButton;
+    public ImageView infoButton;
```

Comment:
Make those members package private, so that you can access them in the test, instead of public.

A member is package private when it has no visibility modifier such as "private" or "public" or "protected".